op-node/withdrawals/utils.go

```diff
@@ -255,14 +263,54 @@ func ParseWithdrawalInitiated(receipt *types.Receipt) (*bindings.L2ToL1MessagePa
 	if err != nil {
 		return nil, err
 	}
-	if len(receipt.Logs) != 1 {
-		return nil, errors.New("invalid length of logs")
+	abi, err := bindings.L2ToL1MessagePasserMetaData.GetAbi()
+	if err != nil {
+		return nil, err
 	}
-	ev, err := contract.ParseWithdrawalInitiated(*receipt.Logs[0])
+
+	for _, log := range receipt.Logs {
+		topic := log.Topics[0]
```

Comment:
can also look at the topic directly, but this does work.

Reply:
updated